src/extension/postgres/mod.rs

```diff
@@ -1,6 +1,7 @@
 pub use expr::*;
 pub use extension::*;
 pub use func::*;
+#[allow(unused_imports)]
```

Comment:
There are no exported members in interval module so clippy warns it, should we just remove this re-export or keep with `#[allow]` for future usages?

Reply:
If that's the case it's good to remove `pub` from `pub use`

Reply:
It seems not used in the module, too.

<img width="959" alt="image" src="https://github.com/SeaQL/sea-query/assets/12772118/123b9991-721c-41d7-8508-d582e12c96f7">

Reply:
You are right, all code in the `interval.rs` is impl so we can remove this `use`.

Reply:
ok, removed entire the re-export in b94ea89